Resources/Prototypes/_LostParadise/Entities/Structures/Wallmounts/halloween.yml

```
@@ -0,0 +1,47 @@
+  description: Picture of some kind of vampire, the sight of him makes you feel cold.
+  components:
+  - type: Sprite
+    sprite: _LostParadise/Structures/Wallmounts/halloween.rsi
+    state: сounts_painting
+    noRot: true
+
+- type: entity
+  parent: BaseSign
+  id: LPPPCheckboxes
+  name: checkboxes
+  description: Black and orange flags, it's got a halloween feel to it. 
```

yamllint

[error] trailing spaces

(trailing-spaces)